Mathlib/Topology/Separation.lean

```diff
@@ -1330,6 +1330,12 @@ theorem Continuous.closedEmbedding [CompactSpace α] [T2Space β] {f : α → β
   closedEmbedding_of_continuous_injective_closed h hf h.isClosedMap
 #align continuous.closed_embedding Continuous.closedEmbedding
 
+/-- A surjective continuous map from a compact space to a Hausdorff space is a quotient map. -/
```

Comment:
nit: the lemma above says "A continuous injective map" in mathlib4 and "An injective continuous map" in mathlib3. Can you make the two lemmas match, either by using the mathlib3 or mathlib4 spelling in both places?